PubSub/src/Topic.php

```diff
@@ -49,16 +59,28 @@ class Topic
 {
     use ArrayTrait;
     use ResourceNameTrait;
+    use ApiHelperTrait;
 
     const DEFAULT_COMPRESSION_BYTES_THRESHOLD = 240;
 
     const DEFAULT_ENABLE_COMPRESSION = false;
 
+    private const COMPRESSION_HEADER_KEY = 'grpc-internal-encoding-request';
+
+    private const GZIP_COMPRESSION = 'gzip';
+
     /**
-     * @var ConnectionInterface A connection to the Google Cloud Platform API
+     * @var RequestHandler
      * @internal
+     * The request handler that is responsible for sending a request and
+     * serializing responses into relevant classes.
+     */
+    private $requestHandler;
```

Comment:
we should add typehints for all new properties
```suggestion
    private RequestHandler $requestHandler;
```